framework/source/class/qx/core/Object.js

```diff
@@ -230,15 +230,31 @@ qx.Class.define("qx.core.Object",
     /**
      * Load user defined data from the object
      *
-     * @param key {String} the key
+     * @param key {String?} the key
+     *   If key is not a string, pass the user the entire map
+     * @param bCopy {Boolean?}
+     *   If bCopy is truthy pass the user a deep clone
      * @return {Object} the user data
```

Comment:
Can you update the documentation of the return value? Something like
"Returns null if no user data exists or if the given key has not been set, the user data map as an object if requested, or the value of the user data with the given key"
either in the JSDoc header or the `@return` tag.

Reply:
updated as requested